docs/articles/blog/2020-7-15-introducing-the-testcafe-jenkins-plugin.md

```diff
@@ -0,0 +1,170 @@
+
+```js
+{
+    "browsers": [
+        "chromium:headless:emulation:device=iPhone 4",
+        "chromium:headless:emulation:device=iPad Mini",
+        "chromium:headless"
+    ],
+    "src": "test/e2e/**/*",
+    "screenshots": {
+        "takeOnFails": true,
+        "fullPage": true
+    },
+    "videoPath": "videos",
+    "reporter": {
+        "name": "jenkins",
+        "output": "report.xml"
+    },
+    "appCommand": "node server.js"
+}
+```
+
+The *Browser* array contains the list of browsers TestCafe will use to run our test. In it, we specified the mobile devices we want Chromium to emulate.
+
+The *name* property of the *reporter* object is set as *jenkins*, which ensures that the reports generated by TestCafe can be properly parsed by the server.
+
+If you decide against using a config file, don't forget to [manually set](https://devexpress.github.io/testcafe/documentation/reference/command-line-interface.html#-r-nameoutput---reporter-nameoutput) the report format when launching your tests, like so:
+
+```bash
+testcafe chrome test.js -r jenkins:report.xml.
+```
+
+Otherwise, just run the following command:
+
+```bash
+npx testcafe
+```
+
+### Step 5. Configure the reporter
+
+Add the "Publish JUnit test result report" *post-build action* to turn the pre-included [JUnit](https://plugins.jenkins.io/junit/) reporter plugin on. To display screenshots and videos alongside your test results, select the "Include links to TestCafe artifacts" option from the "Additional test report features" drop-down.
+
+Important: Don't forget to check the "Retain long standard output/error" box. Otherwise the plugin will automatically truncate information concerning test screenshots and videos.
+
+![JUnit reporter configuration screen](/testcafe/images/jenkins-plugin/junit-reporter-configuration.png)
+
+### Step 6. Run the build
+
+Save the changes to your project and run the build by pressing the "Build now" button on the project page:
+
+![Jenkins Project menu with the Build Now Button highlighted](/testcafe/images/jenkins-plugin/build-now-button.png)
+
+## Looking at the results
+
+After the tests are completed, press this link on the build page to view the results:
+
+![Test results link](/testcafe/images/jenkins-plugin/test-results-link.png)
+
+There you'll see links to screenshots and videos taken during the test.
+
+![Test results view](/testcafe/images/jenkins-plugin/test-results-view.png)
+
+If you were working on fixing the cart button, these artifacts would instantly indicate if it's there or not.
+
+## Jenkins Pipeline integration
+
+If you don't want to use the Jenkins GUI to launch these tests, you can always make use of the [Jenkins Pipeline](https://www.jenkins.io/doc/book/pipeline/). It lets you declaratively configure your continuous delivery setup via Jenkinsfile, a text file checked into the project's source control repository. Here's a sample Jenkinsfile to be used with TestCafe:
```

Comment:
```Jenkinsfile``` can be used not only for ```continuous delivery```, but also for ```continuous integration``` setup (```CI/CD Pipeline```).
I feel that mentioning only ```continuous delivery``` may confuse a reader.